lib/authc/OAuthBasicExchangeAuthenticator.js

```diff
@@ -67,20 +67,23 @@ OAuthBasicExchangeAuthenticator.prototype.buildTokenResponse = function buildTok
 
 OAuthBasicExchangeAuthenticator.prototype.buildAccesstoken = function buildAccesstoken(account) {
   var self = this;
-
   var now = nowEpochSeconds();
+
   var _jwt = {
     sub: self.id,
     iss: self.application.href,
     iat: now,
     exp: now + self.ttl
   };
+
   var scope = self.scopeFactory(account,self.requestedScope);
+
   if(scope){
     // TODO v1.0.0 - remove string option, should be array only
     _jwt.scope = Array.isArray(scope) ? scope.join(' ') : scope;
   }
-  return self._token = jwt.encode(_jwt,self.application.dataStore.requestExecutor.options.apiKey.secret,'HS256');
+
+  return self._token = jwt.encode(_jwt, self.application.dataStore.requestExecutor.options.client.apiKey.secret, 'HS256');
```

Comment:
`self.application.dataStore.requestExecutor.options.client.apiKey.secret`

Dayum, that path!